S/slow5lib/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "slow5lib"
+version = v"1.3.1"
+
+sources = [
+    GitSource("https://github.com/hasindu2008/slow5lib.git", "050c09868696d98735152bf0abb3de766f5ab5cb"),
+    DirectorySource("./patches")
+]
+
+script = raw"""
+if [[ "${target}" == *-apple-darwin* ]]; then
+    actual_lib_dir="${WORKSPACE}/$(readlink ${WORKSPACE}/destdir)/lib"
+    export LDFLAGS="-L${actual_lib_dir} -lz"
```

Comment:
That's really strange. What is the error you get without this?

Reply:
```suggestion
    export LDFLAGS="-lz"
```
Based on the error, isn't this enough?